core-rust/core-api-server/src/core_api/generated/models/sbor_format_options.rs

```diff
@@ -17,7 +17,7 @@ pub struct SborFormatOptions {
     /// Whether to return the raw hex-encoded bytes (default true)
     #[serde(rename = "raw", skip_serializing_if = "Option::is_none")]
     pub raw: Option<bool>,
-    /// Whether to return the programmatic json format (default true)
+    /// Whether to return the programmatic json format (normally default true, defaults false for streamed transactions)
```

Comment:
The only improvement (over this "default is X except when it's not") that I can think of is to put some extra note in the /transactions/stream endpoint's docs (not sure where), saying "unlike others, this endpoint does not output jsons by default".